msal4j-sdk/src/samples/msi-sample-jar/src/main/java/com/microsoft/msi/App.java

```diff
@@ -0,0 +1,112 @@
+package com.microsoft.msi;
+
+import com.microsoft.aad.msal4j.IAuthenticationResult;
+import com.microsoft.aad.msal4j.ManagedIdentityApplication;
+import com.microsoft.aad.msal4j.ManagedIdentityId;
+import com.microsoft.aad.msal4j.ManagedIdentityParameters;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+/**
+ * Sample app to test MSI using a single jar.
+ * To create a jar run mvn install -Dmaven.test.skip=true for msal4j pom and then msi-sample-jar pom.
+ * Copy the jar with dependencies and run on a VM or cloud shell using java -jar msi-sample-jar-1.0.1-jar-with-dependencies.jar
+ */
+public class App 
```

Comment:
This is for our own manual testing so the name doesn't matter to much, but it should be something more informative. Something like "MsiTestApp" if it's going to cover all the MSI scenarios, or "XYZTestApp" if it's only meant to test certain sources.